src/ngraph_deassign_clusters.cc

```diff
@@ -57,19 +57,58 @@ static void MaybeLogPlacement(const Graph* graph) {
   if (!config::IsLoggingPlacement()) return;
 
   std::map<int, std::set<const Node*>> final_cluster_map;
-
+  int number_of_nodes = 0, nodes_marked_for_clustering = 0,
+      nodes_assigned_cluster = 0;
   for (auto node : graph->nodes()) {
+    number_of_nodes++;
+    // Check marked for clustering
+    if (NodeIsMarkedForClustering(node)) {
+      nodes_marked_for_clustering++;
+    }
+
+    // Check Cluster Assignment
     int cluster_idx;
     if (!GetNodeCluster(node, &cluster_idx).ok()) {
       cluster_idx = -1;
+    } else {
+      nodes_assigned_cluster++;
     }
     final_cluster_map[cluster_idx].insert(node);
   }
 
+  int perc_marked_for_clustering_of_total =
+      (int)(((float)nodes_marked_for_clustering) * 100.0 / number_of_nodes);
+  int perc_assigned_clusters_of_total =
+      (int)(((float)nodes_assigned_cluster) * 100.0 / number_of_nodes);
+  int perc_assigned_clusters_of_marked =
+      nodes_marked_for_clustering > 0
+          ? (int)(((float)nodes_assigned_cluster) * 100.0 /
+                  nodes_marked_for_clustering)
+          : 0;
+
+  std::cout << "Number of nodes in the graph: " << number_of_nodes << std::endl;
```

Comment:
Do we want to print these logs all the time?

Reply:
`MaybeLogPlacement` is independent of log levels. hence the `cout` instead of VLOG.

however, we could have 
```
config::IsLoggingPlacement() || log_level >=1
```
in which case `MaybeLogPlacement` is activated through both NGRAPH_TF_LOG_PLACEMENT or VLOG levels. This may or may not be in the scope of this PR

Reply:
@kanvi-nervana  If we use NGRAPH_VLOG instead of cout then we will need both NGRAPH_TF_LOG_PLACEMENT and NGRAPH_TF_VLOG_LEVEL set to see the prints. The couts are consistent with the current implementation of NGRAPH_TF_LOG_PLACEMENT/MaybeLogPlacement(). 

@sayantan-nervana Good Suggestion. Though beyond this PR. I am not sure how cluttered we want our NGRAPH_TF_VLOG logs to be. This function does create a lot of logs.  Will wait for @avijit-nervana s comments.

Reply:
We want these logs to go to stdout. If we use vlog - then there are lots of file/timestamp information that get prefixed making it difficult to parse/understand. Also - the nGraph op placement information may be intermixed with other log items - which is not desired either.